testing/common.test.js

```diff
@@ -54,6 +54,47 @@ const execTestCafeCode = (t, code) => {
   return testCafeFunction(t);
 };
 
+const SKIP_ACCESSIBILITY_TESTS = ['TabPanel-Overview'];
```

Comment:
This exception and cases below will be processed separetelly - we'll remove this skips after fixes.